### DISCUSSION TITLE
PCAP storage rollover to second disk?

### DISCUSSION THREAD

**Discussion by RollingMoses** [Security-Onion-Solutions/securityonion#2292] [Unsupported Versions] [answered]
Very green SO user here researching distributed deployment. Looking at a forward sensor option with 2 physical disks, one SSD, one HDD. Is there a way to set steno to write PCAP data to the SSD, then "archive" the old data to the HDD when that directory is getting full? Or is the only option to set it to write the data to the HDD from the start?

**Comment by weslambert** [comment, marked as answer]
Any PCAP rollover/archival would need to be handled by an external process, whether it be a script, or some other mechanism.
